app/models/form526_submission.rb

```diff
@@ -46,13 +46,37 @@ class Form526Submission < ApplicationRecord
   BIRLS_KEY = 'va_eauth_birlsfilenumber'
   SUBMIT_FORM_526_JOB_CLASSES = %w[SubmitForm526AllClaim SubmitForm526].freeze
 
+  def start
+    if single_issue_hypertension_claim? && Flipper.enabled?(:disability_hypertension_compensation_fast_track)
+      workflow_batch = Sidekiq::Batch.new
+      workflow_batch.on(
+        :success,
+        'Form526Submission#start_evss_submission',
+        'submission_id' => id
+      )
+      jids = workflow_batch.jobs do
+        FastTrack::DisabilityCompensationJob.perform_async(id, get_full_name)
+      end
+      jids.first
```

Comment:
Does this mean that this method is blocked/waiting while the jobs complete?

Reply:
I don't think the method should be blocked or waiting, no. We have the Batch callback that would call the `start_evss_submission` method after our job is complete (or ended in error). That being said, @NatashaJIbrahim knows more about Sidekiq-related topics than I do, so I defer to her.

Reply:
@ericboehs nope. It's not blocked because our code keeps running while the worker is off doing it's own thing. 

BUT this code and the code in the original `Form526Submission#start` method is super weird to me. For now we're intentionally accepting the original inputs for the old `Form526Submission#start` method and having the same output and I think we should keep it that way since our new `#start` is wrapping the old one (which is now `#start_evss_submission`) essentially.

That being said, a good follow up to this pr might be to try and figure out why `Form526Submission#start` was ever returning the first letter of the jid to begin with and if it really needs to return anything since `perform_async` returns the jid anyway 🤷 . I'd rather not introduce any changes to that here though because it feels a bit of scope.

Thoughts? Does that answer your question?

Reply:
`Form526Submission#start_evss_submission` does get blocked by our worker though but we've limited the length of time the worker can attempt to do it's thing to about 4-5 hours before it gives up and kicks the work to `Form526Submission#start_evss_submission`. At that point it shouldn't the submission should continue and be able to complete successfully regardless of whether our worker was successful or not.